lib/bugsnag/report.rb

```diff
@@ -55,6 +56,7 @@ def initialize(exception, passed_configuration, auto_notify=false)
       self.breadcrumbs = []
       self.delivery_method = configuration.delivery_method
       self.hostname = configuration.hostname
+      self.runtime_versions = configuration.runtime_versions
```

Comment:
This should probably be assigned a duplicate (`configuration.runtime_versions.dup`) instead, especially if it is mutable via the `Report` that is yielded in `Bugsnag.notify` (because if for some reason I feel like writing something like `report.runtime_versions['something'] = '1.2.3'`, I shouldn't expect it to change the Bugsnag configuration).

Reply:
Seems reasonable - I've applied this to the other fields taken from `configuration` and assigned to object variables in `init`.